app/driver.go

```diff
@@ -140,10 +153,13 @@ func deployDriver(c *cli.Context) error {
 	switch driver {
 	case FlagDriverCSI:
 		logrus.Debug("Deploying CSI driver")
-		err = deployCSIDriver(kubeClient, c, managerImage)
+		err = deployCSIDriver(kubeClient, c, managerImage, managerURL)
 	case FlagDriverFlexvolume:
 		logrus.Debug("Deploying Flexvolume driver")
 		err = deployFlexvolumeDriver(kubeClient, c, managerImage)
+		if err == nil {
+			err = startProvisioner(kubeClient, managerURL)
```

Comment:
`startProvisioner` won't be executed because `deployFlexvolumeDriver` is a run to complete function. It will block till the end of the process.

We need to run `startProvisioner` first.